### ISSUE TITLE
Keep downloaded artifacts

### ISSUE THREAD

**Issue by alagane** [petalslink/petals-esb-deployer#12]
As a user
I want the downloaded files to be kept somewhere in a dedicated directory
So I don't have to download them multiple times
And I know where the downloaded files are

**Comment by cdeneux** [comment]
If you want to add such a feature, you just manage changes on a remote artefacts already downloaded.
IMO, this is not a requested feature or is not the right one. Can you explain what you have in your mind ?

**Comment by alagane** [comment]
For now, downloaded artifacts (with URL) are in a temporary directory, and can be deleted. The directory where they are stored could be chosen, then it would not be cleaned.

**Comment by cdeneux** [comment]
You are right about configuration of the directory. This should be a dedicated issue.

But why to not clean it ? Isn't it a simple file cache area ?
